src/Sulu/Bundle/AdminBundle/Resources/js/containers/FormOverlay/FormOverlay.js

```diff
@@ -34,12 +34,6 @@ class FormOverlay extends React.Component<Props> {
 
     @observable formErrors: Array<string> = [];
 
-    @computed get confirmDisabled() {
-        const {confirmDisabled, formStore} = this.props;
-
-        return confirmDisabled || !formStore.dirty;
```

Comment:
i figured that disabling the confirm button while the formstore is not empty is not the correct behaviour in every case. eg. wen restoring a category, i want to be able to confirm the overlay without selecting a parent category. 

because of this, i think we should remove this here. if we need this behaviour somewhere, we can still set the `confirmDisabled` prop from the outside.

Reply:
I'm not sure if this has side effects somewhere

Reply:
the `FormOverlay` is used in the `ProfileFormOverlay`, `FormOverlayList` and for displaying the settings overlay of the `FieldBlocks`. i will set the `confirmDisabled` prop in these components to keep the olde behaviour 🙂